widgets/blueprintSources/src/BlueprintSources.tsx

```diff
@@ -107,10 +153,11 @@ export default function BlueprintSources({ data, toolbox, widget }) {
                 <SplitterLayout
                     primaryIndex={0}
                     percentage
-                    secondaryInitialSize={widget.configuration.contentPaneWidth}
+                    secondaryInitialSize={widget.configuration.contentPaneWidth as number}
                 >
                     <div>
                         <NodesTree className="nodes-tree" showLine selectable defaultExpandAll onSelect={selectFile}>
+                            {/* @ts-ignore Missing the following properties from type 'TreeNodeProps': style, onSelect, icon, switcherIconts */}
```

Comment:
As discussed [here](https://github.com/cloudify-cosmo/cloudify-stage/pull/2046#discussion_r915911542)

```suggestion
                            {/* @ts-ignore NodesTree.Node is not migrated to typescript */}
```

Reply:
addressed it here: https://github.com/cloudify-cosmo/cloudify-stage/pull/2046/commits/3d2833b9ddcbdc0958510f7563fb53a463d6e86b